hw7/functional.py

```diff
@@ -0,0 +1,58 @@
+def sequential_map(*args):
+    container = list(args[-1])
+    functions = args[0:(len(args)-1)]
```

Comment:
А вообще удобнее всего сделать так
```suggestion
    *functions, container = args
```